app-backend/tile/src/main/resources/application.conf

```diff
@@ -12,6 +12,17 @@ tile-server {
   cache.enabled = ${?TILESERVER_CACHE_ENABLED}
 }
 
+akka {
+  blocking-scene-routes-dispatcher {
```

Comment:
You're correct: https://github.com/akka/akka/blob/master/akka-actor/src/main/resources/reference.conf#L508 

I think it makes sense only if we want to tune this thread pool for our needs, but as a first solution we can use the default one.